models/testdevice-2.0.x/openapi.yaml

```diff
@@ -317,6 +317,22 @@ components:
       pattern: '[A-Z]{3}-[0-9]*'
       title: leafAtTopLevel
       type: string
+    LeafRefOptions:
```

Comment:
It's kind of un-necessary when there are no leaf refs in the module

Reply:
@SeanCondon Sure, I'll fix it. I won't merge this pr for now, there are other information I would need for leafref, so I might change the openapi-gen again.